pkg/cmd/server/api/v1/types_test.go

```diff
@@ -351,7 +353,8 @@ func TestMasterConfig(t *testing.T) {
 		AssetConfig: &internal.AssetConfig{
 			Extensions: []internal.AssetExtensionsConfig{{}},
 		},
-		DNSConfig: &internal.DNSConfig{},
+		DNSConfig:             &internal.DNSConfig{},
+		AdmissionPluginConfig: map[string]string{"pluginName": "configFilePath"}, // Must be specified explicitly becase it's omitempty
```

Comment:
+1 to a unified structure which expressed both what plugins to load and where their configs are located.